# Query for the meshes of specific entities
string[] entity_ids
---
string[] entity_ids
Mesh[] meshes

# Empty if no errors
string error_msg
